aws-rds-integration/aws-rds-integration.json

```diff
@@ -0,0 +1,155 @@
+      "type": "string",
+      "description": "The ARN of the integration."
+    },
+    "KMSKeyId": {
+      "type": "string",
+      "description": "An optional AWS Key Management System (AWS KMS) key ARN for the key used to to encrypt the integration. The resource accepts the key ID and the key ARN forms. The key ID form can be used if the KMS key is owned by te same account. If the KMS key belongs to a different account than the calling account, the full key ARN must be specified. Do not use the key alias or the key alias ARN as this will cause a false drift of the resource."
+    },
+    "AdditionalEncryptionContext": {
+      "$ref": "#/definitions/EncryptionContextMap"
+    },
+    "CreateTime": {
+      "type": "string"
+    }
+  },
+  "required": [
+    "SourceArn",
+    "TargetArn"
+  ],
+  "definitions": {
+    "Tags": {
+      "type": "array",
+      "maxItems": 50,
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "description": "An array of key-value pairs to apply to this resource.",
+      "items": {
+        "$ref": "#/definitions/Tag"
+      }
+    },
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The key name of the tag. You can specify a value that is 1 to 128 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "description": "The value for the tag. You can specify a value that is 0 to 256 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key"
+      ]
+    },
+    "EncryptionContextMap": {
+      "type": "object",
+      "patternProperties": {
+        "^[\\s\\S]*$": {
+          "type": "string",
+          "maxLength": 131072,
+          "minLength": 0
+        }
+      },
+      "description": "An optional set of non-secret key\u2013value pairs that contains additional contextual information about the data.",
+      "additionalProperties": false
+    }
+  },
+  "propertyTransform": {
+    "/properties/SourceArn": "$lowercase(SourceArn)",
+    "/properties/KmsKeyId": "$join([\"arn:(aws)[-]{0,1}[a-z]{0,2}[-]{0,1}[a-z]{0,3}:kms:[a-z]{2}[-]{1}[a-z]{3,10}[-]{0,1}[a-z]{0,10}[-]{1}[1-3]{1}:[0-9]{12}[:]{1}key\\/\", KmsKeyId])"
+  },
+  "createOnlyProperties": [
+    "/properties/SourceArn",
+    "/properties/TargetArn",
+    "/properties/KMSKeyId",
+    "/properties/AdditionalEncryptionContext",
+    "/properties/IntegrationName"
```

Comment:
For now, I marked `IntegrationName` as createOnly since it cannot be edited at the moment.